ecl/eclagent/eclagent.hpp

```diff
@@ -25,6 +25,8 @@ version: "1.0"
     name: myeclagent
     thorConnectTimeout: 600
     traceLevel: 0
+    costs:
+      workflowMachine: 0.0
```

Comment:
Should this default to non-zero?

Reply:
This file was deleted in HPCC-23609 and the yaml definition moved to ecl/agentexec/agentexec.cpp.